### PR TITLE
2021-12-29-classifierdl_xlm_roberta_sentiment_sw

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/murat-gunay/2021-12-29-classifierdl_xlm_roberta_sentiment_sw.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+sentimentClassifier = ClassifierDLModel.pretrained("classifierdl_xlm_roberta_sentiment", "sw") \
+  .setInputCols(["document", "sentence_embeddings"]) \
+  .setOutputCol("class")
+
+sw_pipeline = Pipeline(stages=[document_assembler, tokenizer, normalizer, stopwords_cleaner, embeddings, embeddingsSentence, sentimentClassifier])
+
+light_pipeline = LightPipeline(sw_pipeline.fit(spark.createDataFrame([['']]).toDF("text")))
+
+result1 = light_pipeline.annotate("Nimechanganyikiwa sana siku hizi.")
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-29-classifierdl_xlm_roberta_sentiment_sw.md:72]: I haven't found in the Internet these sentences, what very often means that they are not 100% native language, that is a translation. Can you use real sentences from the dataset? https://github.com/Jinamizi/Swahili-sentiment-analysis

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Several changes required, take a look. Thank you!

File: docs/_posts/murat-gunay/2021-12-29-classifierdl_xlm_roberta_sentiment_sw.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-12-29-classifierdl_xlm_roberta_sentiment_sw.md]: @josejuanmartinez 
I changed the sentences

File: docs/_posts/murat-gunay/2021-12-29-classifierdl_xlm_roberta_sentiment_sw.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model identifies the sentiments (positive or negative) in Swahili texts.
```
[the remark this answers is not in the document]
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-12-29-classifierdl_xlm_roberta_sentiment_sw.md]: @josejuanmartinez

fixed

**Review by josejuanmartinez — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**